requirements.txt

```diff
@@ -3,7 +3,7 @@
 # http://datastax.github.io/python-driver/installation.html#cython-based-extensions
 futures
 six
--e git+https://github.com/datastax/python-driver.git@cassandra-test#egg=cassandra-driver
+-e git+https://github.com/datastax/python-driver.git@master#egg=cassandra-driver
```

Comment:
You should make sure the changes you need are moved into cassandra-test, rather than switching us to master.

Reply:
By now it's already merged, i needed this one to run tests.